Makefile

```
@@ -33,7 +35,14 @@ azure-upload-preview: build
 	az storage blob upload-batch \
 		-d ${AZURE_STORAGE_CONTAINER} \
 		--account-name ${AZURE_STORAGE_ACCOUNT} \
-		-s ${RELEASE_DIR} --destination-path ileap/$(shell date -u "+%Y%m%dT%H%M%SZ")
-
+		-s ${RELEASE_DIR} --destination-path ileap/${DATE}${HOUR} \
+
+release: clean build
+	$(eval YEAR := $(shell date -u "+%Y"))
+	$(eval DATE := $(shell date -u "+%Y%m%d"))
+	mkdir -p TR/$(YEAR)/ileap-extension-$(DATE)/diagrams
+	sed -E -i '' 's|<h2 id="subtitle" class="no-num no-toc no-ref">.*</h2>|<h2 id="subtitle" class="no-num no-toc no-ref">Stable Release</h2>|' specs/header.include
+	cp specs/index.html specs/faq.html TR/$(YEAR)/ileap-extension-$(DATE)/
+	cp specs/diagrams/*.svg TR/$(YEAR)/ileap-extension-$(DATE)/diagrams/
 
 .PHONY: serve build azure-upload-preview clean
```

```suggestion
.PHONY: serve build azure-upload-preview clean release
```

The `release` rule should be always built as well